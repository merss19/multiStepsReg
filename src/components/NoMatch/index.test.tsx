import {shallow} from 'enzyme';
import {NoMatch} from './index';
import * as React from 'react';
import toJson from 'enzyme-to-json';
import { configure } from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Component Loader', () => {
	const component = shallow(<NoMatch />);
	it('renders component', () => {
		expect(component).toHaveLength(1);
	});
	it('renders correctly', () => {
		const tree = toJson(shallow(<NoMatch />));
		expect(tree).toMatchSnapshot();
	});
});
