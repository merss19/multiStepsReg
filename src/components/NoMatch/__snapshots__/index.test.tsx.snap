// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Loader renders correctly 1`] = `
<div
  className="entry entry--min"
>
  <div
    className="entry__inner text-center "
  >
    <h2
      className="pt10 pb10"
    >
      Ничего не найдено
    </h2>
    <Component
      prefix="btn"
      styleBtn="secondary"
      type="button"
      wide={false}
    >
      <Link
        replace={false}
        to="/"
      >
        Продолжить
      </Link>
    </Component>
  </div>
</div>
`;
